### PR TITLE
Added blueprints-provider

### PR DESCRIPTION
**Pull request by mpastorek**
### Code of Conduct
- [ ] Title of the PR starts with chart name (e.g. `[inventory]`)
- [ ] Update package version in principles of semantic versioning
- [ ] Check RELEASE.md 
- [ ] Update CHANGELOG.md

### Type of Change
- [ ] **Feature**: A new addition to the project that requires a MINOR version bump.
- [ ] **Fix**: A bug fix that requires a PATCH version bump.
- [ ] **Breaking Change**: Any change that is NOT backward-compatible, requiring a MAJOR version bump.

### Versioning Strategy
Explain how the version number was determined based on semantic versioning principles.

### References
- [Semantic Versioning 2.0.0](https://semver.org/)

### CODE REVIEW

**Review by jbogusciak — APPROVED** [review]

**Comment by mpastorek** [comment]
WIP

File: performance-monitor/python/pyproject.toml [hunk 1]
```diff
@@ -5,7 +5,6 @@ build-backend = "poetry.core.masonry.api"
 [tool.poetry.dependencies]
 python = "^3.10"
 pydantic = "^2"
-graphql-pydantic-converter = "^1.2"
```
Comment (Jozefiel) [inline performance-monitor/python/pyproject.toml:8]: why is this removed?

File: performance-monitor/python/frinx_api/performance_monitor/__init__.py [hunk 2]
```diff
@@ -1,229 +1,214 @@
+# generated by datamodel-codegen:
+#   filename:  <stdin>
+#   timestamp: 2024-12-02T13:15:26+00:00
+
```
Comment (Jozefiel) [inline performance-monitor/python/frinx_api/performance_monitor/__init__.py:4]: This is not needed here
